softgroup/services/profile/profile-impl/src/main/java/com/softgroup/profile/impl/handler/GetMyProfileHandler.java

```diff
@@ -17,12 +23,37 @@
 public class GetMyProfileHandler
         extends AbstractRequestHandler<GetMyProfileRequest,
             GetMyProfileResponse> implements ProfileRequestHandler {
+
+    @Autowired
+    private ProfileService profileService;
+
+    @Autowired
+    private Mapper maper;
+
     public String getName() {
         return "get_my_profile";
     }
 
     @Override
     public Response<GetMyProfileResponse> doHandle(Request<GetMyProfileRequest> request) {
-        return null;
+        GetMyProfileRequest requestData = request.getData();
+        GetMyProfileResponse getMyProfileResponse = new GetMyProfileResponse();
+
+        String profileId = request.getRoutingData().getProfileId();
+
+        ProfileEntity profileEntity = profileService.findProfileById(profileId);
+
+        getMyProfileResponse.setProfileDTO((ProfileDTO) maper.map(profileEntity, ProfileDTO.class));
+
+        Response<GetMyProfileResponse> response = new Response<GetMyProfileResponse>();
+        response.setHeader(request.getHeader());
+        response.setData(getMyProfileResponse);
+
+        ResponseStatus status = new ResponseStatus();
```

Comment:
My factory is in the 'newDew'(approved) and in the 'responseFactory' branches) I don't know how can I move it here..))